OrbitService/CMakeLists.txt

```diff
@@ -10,26 +10,31 @@ project(OrbitServiceLib)
 add_library(OrbitServiceLib STATIC)
 target_compile_options(OrbitServiceLib PRIVATE ${STRICT_COMPILE_FLAGS})
 
+target_include_directories(OrbitServiceLib PUBLIC ${CMAKE_CURRENT_LIST_DIR}/include)
+
+target_sources(OrbitServiceLib PUBLIC
+        include/OrbitService/CaptureServiceImpl.h
+        include/OrbitService/CrashServiceImpl.h
+        include/OrbitService/FramePointerValidatorServiceImpl.h
+        include/OrbitService/OrbitGrpcServer.h
+        include/OrbitService/OrbitService.h
+        include/OrbitService/ProcessList.h
+        include/OrbitService/ProcessServiceImpl.h)
```

Comment:
The idea behind having include dir is to place headers that are used by other modules to it. That way we have a separation between public headers and private ones.

Service is not used by anyone so in this case we do not need to have public headers.

Reply:
So are you suggesting to also get rid of the include folder or to just not make them public in the cmake file? 
I like having a separate folder and the prefix in the `#includes`, I think it is more organised and easier to understand where the includes are coming from

Reply:
Since we should only move headers that are supposed to be used from the outside (other modules). I say do not move them to include for the service.

Reply:
done